.github/workflows/ruby.yml

```diff
@@ -0,0 +1,28 @@
+# Run tests!
+name: "Ruby"
```

Comment:
CircleCI test matrix wasn't working correctly. Since I already had GitHub actions ruby matrix working in gl_lint, I copied it over.